package-schema-registry/setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+from pip.req import parse_requirements
+import setuptools
+
+remote_requirements = '\n'.join(str(r.req) for r in parse_requirements("requirements.txt", session='dummy') if r.req)
+
+setuptools.setup(
+    name='cp-schema-registry-tests',
+    version='0.0.1',
```

Comment:
probably doesn't matter much for this particular case, but shouldn't this just match the rest of the version numbers?

Reply:
Yeah -- I think version bumping will catch this, but it would be good to verify. We're not actually publishing the artifacts, but consistency is nice.

Reply:
i think this is true as long as it isn't painful/annoying/difficult. if it requires more than minimal effort, i'm not sure it is worth it in this particular case. just better to be consistent if possible with little to no effort